cmake/scripts/run_test.sh

```diff
@@ -18,143 +18,146 @@
 # ("diff") individual tests.
 #
 # Usage:
-#   run_test.sh run TEST_FULL RUN_COMMAND
+#   run_test.sh run TEST_FULL [COMMAND and ARGS]
 #
-#   run_test.sh diff TEST_FULL DIFF_EXECUTABLE DIFF_EXECUTABLE COMPARISON_FILE
+#   run_test.sh diff TEST_FULL NUMDIFF_EXECUTABLE DIFF_EXECUTABLE \
+#     COMPARISON_FILE
 #
 
 set -u
 
 STAGE="$1"
 TEST_FULL="$2"
-NUMDIFF_EXECUTABLE="$3"
-DIFF_EXECUTABLE="$4"
-COMPARISON_FILE="$5"
-shift 5
-
-save () {
-  for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ;
-  done
-  echo " "
-}
```

Comment:
@tjhei Simply restructuring the shell script and removing this arcane magic is also an option.